pom.xml

```diff
@@ -9,7 +9,7 @@
     </parent>
     <groupId>io.jenkins.plugins</groupId>
     <artifactId>macstadium-orka</artifactId>
-    <version>2.05-SNAPSHOT</version>
```

Comment:
We shouldn't change this manually.
It is automatically updated during the release workflow

Reply:
Understood. I'll. update the development document with that piece of information.

Reply:
Changed to original value and updated Developing.md

Reply:
This is still unchaged

Reply:
@frosemond this is still unchanged. Can we revert it?